features/__snapshots__/validate_input.snap

```diff
@@ -0,0 +1,55 @@
+
+[invalid policy URL:stdout - 1]
```

Comment:
I don't see a scenario for this. Outdated snapshot?

Reply:
Yes. I'll update and create a PR.